helm/values.yaml

```diff
@@ -1,7 +1,7 @@
 # configure Connaisseur deployment
 deployment:
   replicasCount: 3
-  image: securesystemsengineering/connaisseur:v2.4.1
+  image: securesystemsengineering/connaisseur:v2.5.0
```

Comment:
Why is this a MINOR increase? (It's experimental, so a change to the feature would not trigger a MINOR change. Why would its introduction?)

Also, the version change should be a separate commit, so that it can be easily adjusted if the PR gets delayed beyond another release, without touching the feature commit.

Reply:
you are correct. Opened #562 to update versions in accordance with release scope